README.md

```diff
@@ -1,101 +1,110 @@
-![Logo](https://github.com/datatrails/datatrails-scitt-samples/blob/main/DataTrails_Horizontal_Logo_Black.png#gh-light-mode-only)
-![Logo](https://github.com/datatrails/datatrails-scitt-samples/blob/main/DataTrails_Horizontal_Logo_White.png#gh-dark-mode-only)
+![Logo](https://raw.githubusercontent.com/datatrails/datatrails-scitt-samples/main/DataTrails_Horizontal_Logo_Black.png)
```

Comment:
I wasn't sure how these were being used, but they weren't using the blob path to render.